geopandas/geodataframe.py

```diff
@@ -404,6 +460,19 @@ def from_features(cls, features, crs=None, columns=None):
         For more information about the ``__geo_interface__``, see
         https://gist.github.com/sgillies/2217756
 
+        Examples
+        --------
+        >>> feature_coll = {'type': 'FeatureCollection', 'features': [{'id': '0', 'type\
+': 'Feature', 'properties': {'col1': 'name1'}, 'geometry': {'type': 'Point', 'coordinat\
+es': (1.0, 2.0)}, 'bbox': (1.0, 2.0, 1.0, 2.0)}, {'id': '1', 'type': 'Feature', 'proper\
+ties': {'col1': 'name2'}, 'geometry': {'type': 'Point', 'coordinates': (2.0, 1.0)}, 'bb\
+ox': (2.0, 1.0, 2.0, 1.0)}], 'bbox': (1.0, 1.0, 2.0, 2.0)}
```

Comment:
maybe you can try to pretty print this first to use that as a more readable input?